Use DYLD_LIBRARY_PATH instead of LD_LIBRARY_PATH on OS X

OS X does not use LD_LIBRARY_PATH to define additional loader paths. To find
the wkhtmltox library before installation, DYLD_LIBRARY_PATH needs to be set.

Upstream-Status: Submitted [https://github.com/wkhtmltopdf/wkhtmltopdf/pull/3022]
+++ src/image/image.pro.orig	2016-06-28 18:50:52.000000000 +0200
--- src/image/image.pro	2016-06-28 18:51:19.000000000 +0200
@@ -32,6 +32,8 @@
 }
 
 macx {
+    man.extra=DYLD_LIBRARY_PATH=../../bin/ ../../bin/wkhtmltoimage --manpage | gzip > $(INSTALL_ROOT)$$INSTALLBASE/share/man/man1/wkhtmltoimage.1.gz
+
     CONFIG -= app_bundle
 }
 
